Add e2e tests for gb-backend API

ewelinajozwik opened the pull request.

Commit:
Add e2e tests for gb-backend API

Commit:
Improve error response handling for queries endpoint

Commit:
Remove apiVersion from Query table

Commit:
Remove gradleNodeVersion declaration

Commit:
Configure multi project gradle build

It contains single submodule (gb-backend) so far.
gb-backend is excluded as it fails on travis.

Commit:
Add gradle build directory to gitignore

gijskant commented on gb-backend-e2e/build.gradle:
```diff
@@ -0,0 +1,43 @@
+apply plugin: 'java'
+
+ext {
+    jacksonVersion = '2.9.5'
```
Not 2.9.8? (https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-core)

ewelinajozwik replied:
updated!

Commit:
Remove generated gradle wrapper files

in favour of "gradle wrapper" strategy
that requires user to have a gradle installed

Commit:
Update jackson version

gijskant reviewed (approved):
Cool, tests passing, nice to have a light weight dedicated application for this.
